stroom-core-server/src/main/java/stroom/connectors/kafka/StroomKafkaProducerFactoryService.java

```diff
@@ -32,14 +35,14 @@
     private static final String DEFAULT_NAME = "default";
 
     // Old configs may not have named server/version properties, so use this property name
-    private static final String PROP_BOOTSTRAP_SERVERS = "stroom.kafka.bootstrap.servers";
+    private static final String OLD_PROP_BOOTSTRAP_SERVERS = "stroom.kafka.bootstrap.servers";
```

Comment:
kafka was only introduced in v6 which is still unreleased.  I would remove this backwards compatibility